.github/libs/GithubUtils.js

```diff
@@ -221,6 +234,15 @@ class GithubUtils {
                     });
                 }
 
+                if (!_.isEmpty(internalQAPRMap)) {
+                    issueBody += '\r\n\r\n\r\n**Internal QA:**';
+                    _.each(internalQAPRMap, (assignees, URL) => {
+                        const assigneeMentions = _.reduce(assignees, (memo, assignee) => `${memo} @${assignee}`, '');
+                        issueBody += `\r\n\r\n- ${URL} -${assigneeMentions}`;
```

Comment:
Since we only have one checklist per line for QA, let's simplify this a bit:

```suggestion
                        issueBody += `\r\n${_.contains(verifiedOrNoQAPRs, URL) ? '- [x]' : '- [ ]'} ${URL} ${assigneeMentions}`;
```